sdk/videoanalyzer/test-resources.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+    },
+    "variables": {
+    },
+    "resources": [
```

Comment:
For live testing you should probably add something here eventually. I recommend opening an issue. You want to have at least 70% coverage on your SDK, even if it's models only.